packages/web-react/src/components/Toast/README.md

```diff
@@ -315,44 +331,48 @@ This hook returns:
 
 | Name       | Type                                                         | Default    | Description                                         |
 | ---------- | ------------------------------------------------------------ | ---------- | --------------------------------------------------- |
+| `clear`    | `() => void`                                                 | () => {}   | Function that will clear toast queue                |
 | `color`    | [[Emotion Color dictionary][dictionary-color] \| `inverted`] | `inverted` | Color variant                                       |
-| `hide`     | `function`                                                   | () => {}   | Function that will hide UncontrolledToast           |
+| `hide`     | `(toastId) => void`                                          | () => {}   | Function that will hide UncontrolledToast           |
```

Comment:
is this toastId or just the id?

Reply:
hmm, it will be better if i will call it just `id`